### ISSUE TITLE
Separate clean-up and new functionalities in #215

### ISSUE THREAD

**Issue by ineiti** [DP-3T/dp3t-sdk-backend#232]
The current PR #215 has a lot of cleanups that include converting `long` to `UTCInstant`. This hides the original issue described in the PR, so I propose to do the same as for #213, which is to split it into two PRs.

I think one could directly apply to `develop`, while the other probably has to depend on #213.

Making a clean-up PR, the only place I see a _real_ new functionality is https://github.com/DP-3T/dp3t-sdk-backend/pull/215/files#diff-f1905691732404c7d82d000947c50bbeR101

Everything else is just cleaning up.

Also, the `getMaxExposedIdForKeyDateDEBUG` and `getSortedExposedForKeyDateDEBUG` are defined, but never used, and only call the methods without the `DEBUG`.

So perhaps I made a wrong new PR? Or is it really just the new sql-expression in `getMaxExposedIdForKeyDate` and `getSortedExposedForKeyDate`?
